src/themes/journey/theme.json

```diff
@@ -196,31 +196,7 @@
     }
   },
 
-  "accordion": {
-    "base": {
-      "button": {
-        "padding": "sm",
-        "bg": "uswitch-navy",
-        "fontFamily": "base",
-        "fontSize": "base",
-        "fontWeight": "bold",
-        "color": "light-1",
-        "alignItems": "center"
-      },
-      "caret": {
-        "color": "light-1"
-      }
-    },
 
-    "isActive": {
-      "button": {
-        "variant": "accordion.base.button"
-      },
-      "caret": {
-        "color": "light-1"
-      }
-    }
-  },
```

Comment:
Should this be deleted or did you forget to add back in?

Reply:
@barneyfoxuk The latter, have amended!